site/views/examples/share.njk

```diff
@@ -3,7 +3,7 @@ layout: content
 title: Share
 name: Share
 seoTitle: ESL Share - custom element to trigger various share actions for page or url with metadata based on ESL web components
-tags: [examples, beta]
+tags: [examples]
```

Comment:
```suggestion
tags: [examples, playground]
```

Reply:
Little update:  we no longer use playground markers. I hope I'd even remove the renderer for playground